app/templates/tasklist.html

```diff
@@ -112,9 +107,9 @@ <h2 class="govuk-heading-s govuk-!-margin-bottom-2">
                 </h2>
                 <p class="govuk-body govuk-!-margin-bottom-7">
                     {% if application_meta_data["number_of_completed_forms"] == 0 %}
-                        None of the sections have been completed.
+                    None of the sections have been completed.
                     {% else %}
-                        You have completed
+                    You have completed
```

Comment:
Looks good, but are we missing translations for these two lines?

`None of the sections have been completed.`
`You have completed`

Reply:
Oh, bit late just seen this has been merged